core-services/image-mirroring/openshift/mapping_origin_4_10

```diff
@@ -176,6 +177,7 @@ registry.ci.openshift.org/origin/4.10:local-storage-static-provisioner quay.io/o
 registry.ci.openshift.org/origin/4.10:loki quay.io/openshift/origin-loki:4.10 quay.io/openshift/origin-loki:4.10.0 quay.io/openshift/origin-loki:latest
 registry.ci.openshift.org/origin/4.10:machine-api-operator quay.io/openshift/origin-machine-api-operator:4.10 quay.io/openshift/origin-machine-api-operator:4.10.0 quay.io/openshift/origin-machine-api-operator:latest
 registry.ci.openshift.org/origin/4.10:machine-os-content quay.io/openshift/origin-machine-os-content:4.10 quay.io/openshift/origin-machine-os-content:4.10.0 quay.io/openshift/origin-machine-os-content:latest
+registry.ci.openshift.org/origin/4.10:machine-os-images quay.io/openshift/origin-machine-os-images:4.10 quay.io/openshift/origin-machine-os-images:4.10.0 quay.io/openshift/origin-machine-os-images:latest
```

Comment:
Are there multiple images within this image?

Reply:
Currently, no - just the ISO for the current architecture.
In future it's easy to imagine that the baremetal controller would need to deploy workers on multiple architectures, at which point I'd imagine this image would contain an ISO for each supported arch. That was the thought behind the naming.

Reply:
I should add that although there are not two copies, the image can be extracted in two formats (either as an ISO or as kernel/initramfs/rootfs for PXE), and the CoreOS team has plans for being able to generate the other formats without markedly expanding the size. So from the user perspective it can appear to contain multiple images.